github/actions/setupvim/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/govim/govim/githubactions/github/actions/setupvim
```

Comment:
This line, I guess it doesn't matter that much since it isn't imported by anyone. But maybe it's clearer to use something without host to avoid confusion? Like just `setupvim`.

Reply:
If we do that, this then becomes unusable by anyone other than us because the module path (`setupvim`) would not match the address path (`github.com/govim/govim/github/actions/setupvim`). But perhaps you were referring to my typo in the middle of the module path, in which case thanks for the spot :)